index.d.ts

```diff
@@ -1273,6 +1289,10 @@ declare namespace Eris {
       all: 8589934591n;
     };
     REST_VERSION: 8;
+    StickerTypes: {
+      STANDARD: 1,
+      GUILD: 2
+    };
```

Comment:
Don't see these in the constants file

Reply:
Ah, honestly I just noticed `StickerFormats` wasn't in Constants either, so I assumed you didn't want it there. Want me to add both objects while I'm at it?

Reply:
I did that already btw since it made sense to do that ^^ Just lmk if they weren't originally in there for a reason 😄